helm/g2p-sandbox/values.yaml

```diff
@@ -348,7 +349,7 @@ ph-ee-engine:
     imagePullPolicy: "Always"
     SPRING_PROFILES_ACTIVE: "bb"
     LOGGING_LEVEL_ROOT: "INFO"
-    zeebe_broker_contactpoint: "zeebe-zeebe-gateway:26500"
+    zeebe_broker_contactpoint: "g2p-sandbox-zeebe-gateway:26500"
```

Comment:
we can't hardcode this. instead, update this from deployment itself with if the condition
ie, if values are present then take from values, else by default take  {{ .Release.Name }}-zeebe-gateway:26500

Reply:
Fixed in deployment.yaml